docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,78 @@
+"""
+Deriving the Coriolis frequency over the globe
+==============================================
+
+This code computes Coriolis frequency and stores them in a cube with
+associated metadata.
+
+It plots the Coriolis frequency on an orthographic projection.
+
+"""
+
+import cartopy.crs as ccrs
+import iris
+from iris.coord_systems import GeogCS
+import iris.plot as iplt
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def main():
+    # Start with arrays for latitudes and longitudes, with a given resolution.
+    coordinate_points = 200
+    longitudes = np.linspace(-180.0, 180.0, coordinate_points)
+    latitudes = np.linspace(-90.0, 90.0, coordinate_points)
+    lon2d, lat2d = np.meshgrid(longitudes, latitudes)
+
+    # Omega is the Earth's rotation rate, expressed in radians per second
+    omega = 7.29e-5
+
+    # The data for our cube is the Coriolis frequency,
+    # `f = 2 * omega * sin(phi)`, which is computed for each grid point over
+    # the globe from the 2-dimensional latitude array.
+    data = 2. * omega * np.sin(np.deg2rad(lat2d))
+
+    # We now need to define a coordinate system for the plot.
+    # Here we'll use GeogCS; 6371229 is the radius of the Earth in metres.
+    cs = GeogCS(6371229)
+
+    # The Iris coords module turns the latitude list into a coordinate array.
+    # Coords then applies an appropriate standard name and unit to it.
+    lat_coord = iris.coords.DimCoord(latitudes,
+                                     standard_name='latitude',
+                                     units='degrees',
+                                     coord_system=cs)
+
+    # The above process is repeated for the longitude coordinates.
+    lon_coord = iris.coords.DimCoord(longitudes,
+                                     standard_name='longitude',
+                                     units='degrees',
+                                     coord_system=cs)
+
+    # Now we input our data array into the cube.
+    new_cube = iris.cube.Cube(data,
+                              standard_name='coriolis_parameter',
+                              units='s-1',
+                              dim_coords_and_dims=[(lat_coord, 0),
+                                                   (lon_coord, 1)])
+
+    # This section checks to ensure that the coordinates are bounded.
+    # If they have no bounds, the code applies some via guess_bounds()
+    for coord in new_cube.coords():
```

Comment:
@ajdawson okay, thanks for that. I think this approach is a good way to proceed. I've just given this code a local run, and I didn't see any errors or problems, so it looks like the poles aren't introducing any problems.

That said, what I'll also do is restrict the latitude range to (say) -89.5 to 89.5, to avoid the poles entirely. I'd been worried it might create some empty space on the plot, but having run the code with those terms, I'm not seeing any problems.